docs/docs/docs/developer-resources/testing.md

```
@@ -184,6 +185,57 @@
+
+This applies to users running Talawa API in dev containers.
+
+1. Once the server is running, open a new terminal session.
+
+2. Open a bash session inside the running container:
+
+  ```bash
+  docker exec -it talawa-api-1 /bin/bash
+  ```
+
+3. **WARNING:** This command will **DELETE** all data from each table in your database, administrator roles will be restored. Use with extreme caution.
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)